api/src/main/resources/workbench.yaml

```diff
@@ -977,79 +977,81 @@ paths:
           required: true
           description: specifies which workspace
         - in: path
-          name: cohortId
+          name: cohortReviewId
           type: integer
           format: int64
           required: true
-          description: specifies which cohort
+          description: specifies which review
         - in: path
-          name: cdrVersionId
+          name: participantId
           type: integer
           format: int64
           required: true
-          description: specifies which cdr version
+          description: specifies which participant
       responses:
         "200":
-          description: A collection of CohortAnnotationDefinition
+          description: A ParticipantCohortAnnotation.
           schema:
-            $ref: "#/definitions/CohortAnnotationDefinitionListResponse"
+            $ref: "#/definitions/ParticipantCohortAnnotationListResponse"
 
-  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/annotationdefinitions/{annotationDefinitionId}:
+  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortReviewId}/participants/{participantId}/annotations/{annotationId}:
     put:
```

Comment:
Do we need delete support too?

An alternative to this method, FWIW, would be having the POST endpoint above take a list of ParticipantCohortAnnotation, and do insert-or-update logic for each annotation. That way if we ever wanted to allow for setting multiple annotations at a time, you could use the same method. (This seems fine, too.)

Reply:
Good catch on delete. Done. Good idea on the alternative. Currently we don't have a need to edit multiple annotations. I'm fine with changing it if that need surfaces.